services/self-service/src/test/java/com/epam/dlab/backendapi/resources/UserGroupResourceTest.java

```diff
@@ -105,7 +105,7 @@ public void createGroupWhenRoleIdIsEmpty() {
 				.target("/group")
 				.request()
 				.header("Authorization", "Bearer " + TOKEN)
-				.post(Entity.json(getCreateGroupDto(GROUP, Collections.emptySet())));
+				.post(Entity.json(getCreateGroupDto(GROUP, Collections.<String, String>emptyMap())));
```

Comment:
Can we use **Collections.emptyMap()** insead of **Collections.<String, String>emptyMap()**?

Reply:
The collection has been adjusted. Fix has been tested with creation, update and delete group actions